package.json

```diff
@@ -25,7 +25,8 @@
     "react": "15.4.2"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
Just out of curiosity, have you checked how big these are? I played around with lodash modularity a few years ago, and it wasn't that much savings. Hopefully it has improved over the years. I'm fine with whatever you decide here btw, just covering a few bases.